app/webpacker/controllers/flatpickr_controller.js

```diff
@@ -60,6 +60,14 @@ export default class extends Flatpickr {
 
   open() {
     this.fp.element.dispatchEvent(new Event("focus"));
+    if (!this.defaultDateValue) return;
+    if (this.fp.selectedDates.length) return;
```

Comment:
if a date is already selected, we don't need to set a default value

Reply:
👌